src/libpgmoneta/value.c

```diff
@@ -456,14 +460,60 @@ string_to_string_cb(uintptr_t data, int32_t format, char* tag, int indent)
    {
       if (format == FORMAT_JSON || format == FORMAT_JSON_COMPACT)
       {
-         snprintf(buf, MISC_LENGTH, "\"%s\"", str);
+         len = strlen(str);
+         for (int i = 0; i < len; i++)
+         {
+            if (str[i] == '\'' || str[i] == '\"' || str[i] == '\\' || str[i] == '\n' || str[i] == '\t' || str[i] == '\r')
+            {
+               translated_len++;
+            }
+            translated_len++;
+         }
+         translated_ec_string = (char*)malloc(translated_len + 1);
+
+         for (int i = 0; i < len; i++, idx++)
+         {
+            switch (str[i])
+            {
+               case '\\':
+               case '\'':
+               case '\"':
+                  translated_ec_string[idx] = '\\';
+                  idx++;
+                  translated_ec_string[idx] = str[i];
+                  break;
+               case '\n':
+                  translated_ec_string[idx] = '\\';
+                  idx++;
+                  translated_ec_string[idx] = 'n';
```

Comment:
Should we translate this? Perhaps we can just leave it... I remember you said parsing works with char `\n`. Same comment goes to other white space characters.

Reply:
I think we can leave it as long as it can be read or parsed with no ambiguity, we can worry about aesthetic later. Thoughts?

Reply:
It is required while formatting in json as per the specs - https://www.json.org/json-en.html

because if not done so it will treat `\n` as a newline which we don't want in a string according to the specs

Reply:
Ok then, so we are good